src/fpylll/fplll/lll.pyx

```diff
@@ -31,10 +31,13 @@ from fplll cimport FloatType
 from fpylll.util cimport check_float_type, check_delta, check_eta, check_precision
 from fpylll.util import ReductionError
 
-from decl cimport d_t, ld_t
+from decl cimport d_t
 from decl cimport gso_mpz_d, gso_mpz_ld, gso_mpz_dpe, gso_mpz_mpfr
 from decl cimport gso_long_d, gso_long_ld, gso_long_dpe, gso_long_mpfr
 
+IF HAVE_LONG_DOUBLE:
+    from decl import ld_t
```

Comment:
This should read `cimport`; see Travis-CI error

Reply:
Oops I'll fix that.